practice-two/src/index.html

```diff
@@ -126,7 +126,38 @@ <h3 class="advantages__description-label">Unique visitors</h3>
     <!-- END ADVANTAGES SECTION-->
 
     <!-- START TESTIMONIALS SECTION -->
-    <section></section>
+    <section class="testimonials">
+      <div class="container testimonials__wrapper">
+        <h2 class="heading-section text-color-base testimonials__header-text">What Clients are saying</h2>
+        <div class="testimonials__content">
+          <div class="testimonials__description">
+            <p class="text-color-base testimonials__description-text">Businesses at present had creatively generated effective
+              advertising campaigns in the form of audio visual media that includes the television and radios and the print media that cover the newspaper and
+              campaign materials like the posters, brochures, business cards, flyers and many more.
+            </p>
+          </div>
+          <div class="testimonials___info">
+            <div class="testimonials___info-img">
+              <img src="assets/images/person-testinomial.webp" alt="person testimonials img"  class="person-img">
+            </div>
+            <div class="testimonials__contact">
+              <p class="text-color-primary testimonials__contact-name">Ophelia Holland</p>
+              <p class="text-color-base testimonials__contact-mail">@ophelia_holland</p>
+            </div>
+          </div>
+        </div>
+        <a href="javascript:void(0)" class="testimonials__link testimonials__icon-prev">
```

Comment:
To avoid use absolute here. Your layout should be

```
<div class="testimonials__content">
  <a class="testimonials__link testimonials__icon-prev"></a>
  <div>
    <div class="testimonials__description">
      ....
    </div>
    <div class="testimonials___info">

    </div>
  </div>
  <a class="testimonials__link testimonials__icon-next"></a>
</div>
```